web/pdf_page_view.js

```diff
@@ -142,6 +142,25 @@ class PDFPageView {
     }
   }
 
+  /**
+   * @private
+   */
+  async _renderAnnotationLayer() {
+    let error = null;
+    try {
+      await this.annotationLayer.render(this.viewport, "display");
+      error = false;
```

Comment:
Do you actually need this?
Or in other words, why can't the default value (when no error occurred) simply be `null` instead?